mycore-mods/src/main/resources/xslt/solr/indexing/mods-solr.xsl

```diff
@@ -278,7 +278,12 @@
   <xsl:template name="printModsName">
     <xsl:choose>
       <xsl:when test="mods:displayForm">
-        <xsl:value-of select="fn:normalize-unicode(mods:displayForm)" />
+        <xsl:for-each select="mods:displayForm" >
```

Comment:
This can be solved shorter with a String-function:
`<xsl:value-of select="fn:string-join(fn:normalize-unicode(mods:displayForm), ' ')" />`